README.md

```diff
@@ -722,6 +740,20 @@ Server like this:
 
 And then using this code to verify that the topology is the expected one:
 
+#### Note for Mac Users:
```

Comment:
I think it is best to defer to the Infinispan server docs for this kind of info/troubleshooting. /cc @oraNod

Reply:
the problem is developing on mac as well, as well as docs maybe. I rather leave it there as well